Makefile

```diff
@@ -225,9 +225,10 @@ benchmark: all
 
 clean:
 	rm -fr dist/*
-	rm -fr src/*/*.o
-	rm -fr src/*/*.gen.*
 	rm -fr node_modules
+	shopt -s globstar && rm -fr src/**/*.o
```

Comment:
On my Ubtunu  system,
```
zsh: command not found: shopt
```
so it's bash specific. Not sure if it's worth having bash-specific commands here.

Similarly MacOS has zsh so it will not work there.

Reply:
I switched to using `find -delete` which should hopefully work more broadly.